tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -552,7 +552,7 @@ def test_vmgenid(guest_kernel_linux_6_1, rootfs, microvm_factory, snapshot_type)
     platform.machine() != "aarch64" or global_props.host_linux_version_tpl < (6, 4),
     reason="This is aarch64 specific test and should only be run on 6.4 and later kernels",
 )
-def test_physical_couter_reset_aarch64(uvm_nano):
+def test_physical_counter_reset_aarch64(uvm_nano):
```

Comment:
commit message: would be nice to add a link to the upstream patches or refer to our prod host setup doc because it will be hard to find them later on by looking at this commit in isolation.

Reply:
added a link